docs/painless/painless-execute-script.asciidoc

```diff
@@ -0,0 +1,53 @@
+[[painless-execute-api]]
+=== Painless execute API
+
+The Painless execute API allows to execute an arbitrary script and introspect the result.
+
+[[painless-execute-api-parameters]]
+.Parameters
+[options="header"]
+|======
+| Name             | Required  | Default                | Description
+| `script`         | yes       | -                      | The script to execute
+| `context`        | no        | `execute_api_script`   | The context the script should be executed in.
+|======
+
+==== Contexts
+
+Contexts control how scripts get executed, what variables are available at runtime and what the return type is.
```

Comment:
Please replace "get" with "are"

Reply:
Also consider rewording variables into something like parameters?  I realize that's confusing too as params is often used in scripts.  Variables seems to imply what can be locally created as well when really we just want to say what external data is allowed to be used.